src/gui/cookiesdialog.cpp

```diff
@@ -85,5 +87,13 @@ void CookiesDialog::onButtonAddClicked()
 
 void CookiesDialog::onButtonDeleteClicked()
 {
-    m_cookiesModel->removeRow(m_ui->treeView->selectionModel()->currentIndex().row());
+    const QModelIndexList idxs = m_ui->treeView->selectionModel()->selectedRows();
+
+    QVector<int> rows;
+    for (const auto &idx : idxs)
+        rows += idx.row();
+    std::sort(rows.begin(), rows.end(), std::greater<int>());
```

Comment:
Unless I'm wrong, you can sort list of indexes itself and avoid additional array creation.